{{cookiecutter.repo_name}}/.gitignore

```diff
@@ -102,3 +102,6 @@ build.{{ cookiecutter.project_name }}/docker-base/*
 
 # microcosm config
 {{ cookiecutter.project_name }}.conf
+
+[mypy]
```

Comment:
This is the `setup.cfg` config; so it's not what you want here.

Reply:
Oops, will update.